packages/agent/package.json

```diff
@@ -13,10 +13,10 @@
   ],
   "scripts": {
     "lint": "eslint '{app,bin,src,test}/**/*.ts'",
-    "test": "mocha -r ts-node/register test/**/*.test.ts",
+    "test": "TS_NODE_PROJECT=tsconfig.test.json mocha -r ts-node/register test/**/*.test.ts",
```

Comment:
The `yarn test` command works just the same for me if we don't include the `TS_NODE_PROJECT` variable. Seems like this is necessary?

Reply:
@cowboyd it turns out that we don't need any of this. I removed TS_NODE_PROJECT and all of tsconfig.test.json files and tests are running green.